e2e-tests/cypress/support/insurance/complete-company-details-form.js

```diff
@@ -0,0 +1,17 @@
+import { companyDetails } from '../../e2e/pages/your-business';
+
+const completeCompaniesDetailsForm = (companiesHouseNumber, phoneNumber, website) => {
```

Comment:
Can we change both new cypress commands to have the params in an object structure and with documentation? Makes it easier to only change specific values e.g instead of this

```js
cy.completeCompanyDetailsForm(COMPANIES_HOUSE_NUMBER, null, WEBSITE_EXAMPLES.VALID);
```

Would be able to do this:

```js
cy.completeCompanyDetailsForm({ website: WEBSITE_EXAMPLES.VALID });
```